CHANGELOG.rst

```diff
@@ -17,6 +17,11 @@ of the list).
 
 3.0.2 (unreleased)
 ==================
+- Added code to iteratively "grow out" saturated pixels in the DQ binary bitmask [#219]
+
+- Added relative fit source matching algorithm to alignimages. New algorithm currently commented out pending
+  optimization [#212]
```

Comment:
The numbers here are JIRA tickets and not Git PRs, and this could be confusing.  I think this should be made clear in case someone wants to research the background.  I will suggest information, but I am not sure if there is a rule regarding the specific format ==> [JIRA HLA-###].

Reply:
Issue resolved in commit [1a60115](https://github.com/spacetelescope/drizzlepac/pull/268/commits/1a6011588fcb0e0be507fd300dbae721571f06da)